config.go

```diff
@@ -207,21 +208,8 @@ func getConfigPaths() (configPath string, configDirectory string, err error) {
 	}
 
 	// Determine absolute path to config file and directory.
-	configDirectory = filepath.Join(userHome, directoryName)
-	configPath = filepath.Join(userHome, directoryName, fileName)
+	configDirectory = filepath.Join(path, directoryName)
+	configPath = filepath.Join(path, directoryName, fileName)
 
 	return configPath, configDirectory, nil
 }
-
-func getHomeFolder() string {
-	current, e := user.Current()
-	if e != nil {
-		// Give up and try to return something sensible
-		home, err := os.UserHomeDir()
-		if err != nil {
-			return ""
-		}
```

Comment:
This empty return concerned me because we use this value in filepath.Join prior to deleting files.